source/macos/AdaptiveCards/AdaptiveCards/Managers/ACSFillerSpaceManager.swift

```diff
@@ -110,6 +110,23 @@ class ACSFillerSpaceManager {
         }
     }
     
+    /// This function reports if a padding view is already hidden or has been marked to be hidden
+    func isPaddingInvisble(invisibleViews: NSMutableSet) -> Bool {
```

Comment:
I tried to think of a way, but either we pass the invisibleViews to filler space manager or pass the padding views to contentStack View since both are stored individually. Can you suggest something?